### ISSUE TITLE
Unity version 2019.3.1f1 'ReorderableList' does not contain a definition for 'IsSelected'

### ISSUE THREAD

**Issue by BlazingMonk40** [Unity-Technologies/2d-extras#310]
I'm running Unity 2019.3.1f1 and I initially installed the master version of 2d-extras unknowingly. After getting the, "Packages\2d-extras-master\Editor\Tiles\RuleTile\RuleTileEditor.cs(347,82): error CS1061: 'ReorderableList' does not contain a definition for 'IsSelected' and no accessible extension method 'IsSelected' accepting a first argument of type 'ReorderableList' could be found (are you missing a using directive or an assembly reference?)", error message I read the READ-ME and saw I needed the 2019.4 branch. So I uninstalled the package from the package manager and made sure to remove any trace of it in \Packages through File Explorer. And then downloaded the 2019.4 .zip, extracted it to \Packages, and added the json line from the READ-ME to the manifest.json in \Packages and I am still getting the error.

Could someone tell me what I am doing wrong?
Thank you,

**Comment by ChuanXin-Unity** [comment]
Sorry, it seems like switching to the 2019.4 branch and downloading the zip file seems to be downloading the master branch instead right now. I will make a 2019.4 release and post a link you can download from .

**Comment by ChuanXin-Unity** [comment]
Could you try the following link and see if this works for you? Thanks!

https://github.com/Unity-Technologies/2d-extras/archive/refs/tags/1.6.0-preview.1.zip

**Comment by BlazingMonk40** [comment]
Yeah, that seems to have worked, I haven't tried using anything from the package, but I don't have any more errors and I have the 2D tiles and brushes in the Project > Create > tab. Thanks!

**Comment by Retr0-Wolf** [comment]
Thank you this has also fixed it for me
